docs/views/examples/pass-data/index.html

```diff
@@ -81,6 +81,46 @@ <h3 class="govuk-heading-s">
 
       <h2 class="govuk-heading-m">Advanced features</h2>
 
+      <h3 class="govuk-heading-s">
+        Using the data in Nunjucks macros
+      </h3>
+
+      <p>Example using the 'checked' function in a radio component macro:</p>
+
+      <pre class="app-code"><code>{% raw %}
+      {{ govukRadios({
```

Comment:
Can we remove all the indent here? It adds unnecessary indent to the code example, because of the `<pre>`

Reply:
sure, updated